dll/win32/shlwapi/CMakeLists.txt

```diff
@@ -50,6 +51,6 @@ add_dependencies(shlwapi_autocomp psdk)
 set_module_type(shlwapi win32dll UNICODE)
 target_link_libraries(shlwapi uuid wine cpprt)
 add_delay_importlibs(shlwapi userenv oleaut32 ole32 comdlg32 mpr mlang urlmon shell32 winmm version)
-add_importlibs(shlwapi user32 gdi32 advapi32 wininet msvcrt kernel32 ntdll)
+add_importlibs(shlwapi comctl32 user32 gdi32 advapi32 wininet msvcrt kernel32 ntdll)
```

Comment:
NOTE: Delay loading on ordinal function won't work correctly.

Reply:
Won't work correctly where? In MSVC builds? In GCC builds? On both?

Reply:
GCC build.

Reply:
not related to ordinal, but the way the symbol is imported.
https://jira.reactos.org/browse/CORE-6504

Reply:
`comctl32` link is deletedw